<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogStart</class>
 <widget class="QDialog" name="DialogStart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Projektname</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_ProjectName">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_ProjectStatus">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Rohteil">
     <property name="title">
      <string>Rohteil</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_X">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_RTx">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_Y">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_RTy">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_Z">
          <property name="text">
           <string>Z</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_RTz">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_RohteilKontrolle">
          <property name="text">
           <string>Rohteilkontrolle00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_Rohteilkontrolle">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/Icons/more.png</normalon>
           </iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Bauteil">
     <property name="title">
      <string>Bauteil</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_X_2">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_BTx">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_Y_2">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_BTy">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_Z_2">
          <property name="text">
           <string>Z</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_BTz">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_ZRohTeil">
       <property name="text">
        <string>Z Rohteil</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_ZRohTeil">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_Spannung">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Spannung</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QComboBox" name="comboBox_Clamping"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_NP">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_Material">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Material</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_Material"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_AufmassMaxRT">
     <property name="title">
      <string>Aufmass Max vom Rohteil</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="checkBox_RT_EinzelAufmass">
          <property name="text">
           <string>Einzelaufmasse</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_RT_All">
          <property name="text">
           <string>Alle Werte</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_RT_All">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
          <property name="value">
           <double>2.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_RT_AufmassEinzel" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_RT_Xplus">
           <property name="text">
            <string>X+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_Xplus_Max">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
           <property name="value">
            <double>2.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_RT_Xminus">
           <property name="text">
            <string>X-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_Xminus_Max">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
           <property name="value">
            <double>2.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_RT_Yplus">
           <property name="text">
            <string>Y+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_Yplus_Max">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
           <property name="value">
            <double>2.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_RT_Yminus">
           <property name="text">
            <string>Y-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_Yminus_Max">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
           <property name="value">
            <double>2.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_RT_Zplus">
           <property name="text">
            <string>Z+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_Zplus_Max">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
           <property name="value">
            <double>2.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_AufmassMinFT">
     <property name="title">
      <string>Aufmass Min vom Bauteil</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QCheckBox" name="checkBox_FT_EinzelAufmass">
          <property name="text">
           <string>Einzelaufmasse</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_FT_All">
          <property name="text">
           <string>Alle Werte</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_FT_All">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
          <property name="value">
           <double>1.500000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_FT_AufmassEinzel" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_FT_Xplus">
           <property name="text">
            <string>X+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_Xplus_Min">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
           <property name="value">
            <double>1.500000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_FT_Xminus">
           <property name="text">
            <string>X-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_Xminus_Min">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
           <property name="value">
            <double>1.500000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_FT_Yplus">
           <property name="text">
            <string>Y+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_Yplus_Min">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
           <property name="value">
            <double>1.500000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_FT_Yminus">
           <property name="text">
            <string>Y-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_Yminus_Min">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
           <property name="value">
            <double>1.500000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_FT_Zplus">
           <property name="text">
            <string>Z+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_Zplus_Min">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
           <property name="value">
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_ProjectName</tabstop>
  <tabstop>lineEdit_ProjectStatus</tabstop>
  <tabstop>lineEdit_RTx</tabstop>
  <tabstop>lineEdit_RTy</tabstop>
  <tabstop>lineEdit_RTz</tabstop>
  <tabstop>lineEdit_BTx</tabstop>
  <tabstop>lineEdit_BTy</tabstop>
  <tabstop>lineEdit_BTz</tabstop>
  <tabstop>doubleSpinBox_ZRohTeil</tabstop>
  <tabstop>comboBox_Clamping</tabstop>
  <tabstop>lineEdit_NP</tabstop>
  <tabstop>comboBox_Material</tabstop>
  <tabstop>checkBox_RT_EinzelAufmass</tabstop>
  <tabstop>doubleSpinBox_RT_All</tabstop>
  <tabstop>doubleSpinBox_Xplus_Max</tabstop>
  <tabstop>doubleSpinBox_Xminus_Max</tabstop>
  <tabstop>doubleSpinBox_Yplus_Max</tabstop>
  <tabstop>doubleSpinBox_Yminus_Max</tabstop>
  <tabstop>doubleSpinBox_Zplus_Max</tabstop>
  <tabstop>checkBox_FT_EinzelAufmass</tabstop>
  <tabstop>doubleSpinBox_FT_All</tabstop>
  <tabstop>doubleSpinBox_Xplus_Min</tabstop>
  <tabstop>doubleSpinBox_Xminus_Min</tabstop>
  <tabstop>doubleSpinBox_Yplus_Min</tabstop>
  <tabstop>doubleSpinBox_Yminus_Min</tabstop>
  <tabstop>doubleSpinBox_Zplus_Min</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogStart</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>580</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogStart</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>580</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_RT_All</sender>
   <signal>valueChanged(double)</signal>
   <receiver>doubleSpinBox_Xplus_Max</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_RT_All</sender>
   <signal>valueChanged(double)</signal>
   <receiver>doubleSpinBox_Xminus_Max</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_RT_All</sender>
   <signal>valueChanged(double)</signal>
   <receiver>doubleSpinBox_Yplus_Max</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_RT_All</sender>
   <signal>valueChanged(double)</signal>
   <receiver>doubleSpinBox_Yminus_Max</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_RT_All</sender>
   <signal>valueChanged(double)</signal>
   <receiver>doubleSpinBox_Zplus_Max</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_FT_All</sender>
   <signal>valueChanged(double)</signal>
   <receiver>doubleSpinBox_Xplus_Min</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_FT_All</sender>
   <signal>valueChanged(double)</signal>
   <receiver>doubleSpinBox_Xminus_Min</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>517</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_FT_All</sender>
   <signal>valueChanged(double)</signal>
   <receiver>doubleSpinBox_Yplus_Min</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>496</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_FT_All</sender>
   <signal>valueChanged(double)</signal>
   <receiver>doubleSpinBox_Yminus_Min</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>537</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
